openwisp_controller/geo/estimated_location/test_estimated_location.py

```diff
@@ -510,3 +511,251 @@ def test_estimate_location_status_remove(self, mock_client):
+        org1 = self._get_org()
+        org2 = self._create_org(name="org2")
+        OrganizationConfigSettings.objects.create(
+            organization=org2,
+            whois_enabled=False,
+            estimated_location_enabled=False,
+        )
+        org1_location = self._create_location(
+            name="org1-location", organization=org1, is_estimated=True
+        )
+        org2_location = self._create_location(name="org2-location", organization=org2)
+        org1_device = self._create_device(organization=org1)
+        org2_device = self._create_device(organization=org2)
+        self._create_device_location(content_object=org1_device, location=org1_location)
+        self._create_device_location(content_object=org2_device, location=org2_location)
+
+        with self.subTest("Test Estimated Location in Locations List"):
+            path = reverse("geo_api:list_location")
+            with self.assertNumQueries(5):
+                response = self.client.get(path)
+            self.assertEqual(response.status_code, 200)
+            self.assertEqual(response.data["count"], 2)
+            self.assertContains(response, org1_location.id)
+            self.assertContains(response, org2_location.id)
+            location1 = response.data["results"][1]
+            location2 = response.data["results"][0]
+            self.assertIn("is_estimated", location1)
+            self.assertNotIn("is_estimated", location2)
+
+        with self.subTest("Test Estimated Location in Device Locations List"):
+            path = reverse("geo_api:device_location", args=[org1_device.pk])
+            with self.assertNumQueries(4):
+                response = self.client.get(path)
+            self.assertEqual(response.status_code, 200)
+            self.assertIn("is_estimated", response.data["location"]["properties"])
+
+            path = reverse("geo_api:device_location", args=[org2_device.pk])
+            with self.assertNumQueries(4):
+                response = self.client.get(path)
+            self.assertEqual(response.status_code, 200)
+            self.assertNotIn("is_estimated", response.data["location"]["properties"])
+
+        with self.subTest("Test Estimated Location in GeoJSON List"):
+            path = reverse("geo_api:location_geojson")
+            with self.assertNumQueries(3):
+                response = self.client.get(path)
+            self.assertEqual(response.status_code, 200)
+            self.assertEqual(response.data["count"], 2)
+            for i in response.data["features"]:
+                if i["id"] == org1_location.id:
+                    self.assertIn("is_estimated", i["properties"])
+                    self.assertTrue(i["properties"]["is_estimated"])
+                elif i["id"] == org2_location.id:
+                    self.assertNotIn("is_estimated", i["properties"])
+                    self.assertFalse(i["properties"].get("is_estimated", False))
```

Comment:
This catched my attention, why are you providing `False` as fallback value for `get`?

Reply:
I think this might have been left as part of some previous code when I was writing the tests and not using conditionals here. Will remove this